src/OVAL/oval_probe_ext.c

```diff
@@ -1141,13 +1141,14 @@ int oval_probe_ext_abort(SEAP_CTX_t *ctx, oval_pd_t *pd, oval_pext_t *pext)
 	switch (dsc->scheme) {
 	case SCH_PIPE:
 	{
```

Comment:
Those `#if...` blocks make the code harder to understand.

Reply:
OK